Vesper-Agents-Suite/Fund/src/utils.ts

```diff
@@ -0,0 +1,41 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+const defaultList = [
+  "0xcA0c34A3F35520B9490C1d58b35A19AB64014D80",
+  "0x8d0b8e2b5584cE1487317f81Da7d97397eF3e899",
+  "0xa4F1671d3Aee73C05b552d57f2d16d3cfcBd0217",
+  "0x097ee00F42f9D7512929A6434185Ae94aC6dafD7",
+];
+
+const getTotalValue = async (contract: any) => {
+  return await contract.methods.totalValue().call();
+};
+
+const getTokensHere = async (contract: any) => {
+  return await contract.methods.tokensHere().call();
+};
+const getBPSValue = async (contract: any) => {
+  return await contract.methods.MAX_BPS().call();
+};
+const getTotalDebtRatio = async (contract: any) => {
+  return await contract.methods.totalDebtRatio().call();
+};
```

Comment:
The current agent will check for the pool status for every new block, since the function is called whenever a new block is received. Do you think it would be wise to specify a block in this case ? Because we need to deal with the latest status.

Reply:
You have to deal with the block that you are analyzing. There can be a delay and when you receive the block `X`, the block `X + Y` can be already mined so you'll never check `X`.